### DISCUSSION TITLE
京东百里 刷了官方线uboot和immortalwrt，发现wan口经常掉网

### DISCUSSION THREAD

**Discussion by Rem1x-2019** [immortalwrt/immortalwrt#1476] [Q&A]
设备：京东云百里
系统：https://downloads.immortalwrt.org/releases/23.05.3/targets/mediatek/filogic/immortalwrt-23.05.3-mediatek-filogic-jdcloud_re-cp-03-squashfs-sysupgrade.itb
插件：只安装了dae和upnp

1.
![QQ截图20240922121202](https://github.com/user-attachments/assets/2158786a-a05c-4512-958f-8ed50234fd13)
eth1接口经常断开，这个是光猫拨号的，内核日志，
![QQ图片20240922121339](https://github.com/user-attachments/assets/f0273be9-c098-4757-9206-5a7da3a3d8bf)
这个是路由器拨号，系统日志，

无论是ping百度还是光猫，都会时不时超时，就是断开，
换回之前的红米AX6，就没有这个问题。。
路由器是新买的，才到2天。。。

**Comment by 1715173329** [comment]
用 23.05-snapshot 看看

**Reply by Rem1x-2019** [reply]
> 用 23.05-snapshot 看看

暂时没有掉线了，目前是DHCP，路由器拨号，然后LAN1端口没有使用，插到LAN2了，
看来应该是WAN或者LAN口的配置问题。。。

**Comment by Rem1x-2019** [comment]
有个小问题，IMM的分区空间有点小，，才300M左右，，
有没有办法扩大。。我看到有其他大佬有2G的GPT分区，
我现在是官方uboot，能不能只刷2G的GPT分区，然后刷IMM的固件。。。

**Comment by Rem1x-2019** [comment]
然后光猫拨号，LAN2口接电脑，又不行了。。
下面内核日志

[    5.403220] GACT probability on
[    5.407024] Mirror/redirect action on
[    5.414887] u32 classifier
[    5.417596]     input device check on
[    5.421359]     Actions configured
[    5.430327] crypto-safexcel 10320000.crypto: EIP97:230(0,1,4,4)-HIA:270(0,5,5),PE:150/433(alg:7fcdfc00)/0/0/0
[    5.448330] Loading modules backported from Linux version v6.1.102-0-gc1cec4dad96b5
[    5.455980] Backport generated by backports.git v6.1.102-1-0-g40bdd073
[    5.527333] urngd: v1.0.2 started.
[    5.555326] mt798x-wmac 18000000.wifi: attaching wed device 0 version 2
[    5.603070] platform 15010000.wed: MTK WED WO Firmware Version: DEV_000000, Build Time: 20221012175005
[    5.612406] platform 15010000.wed: MTK WED WO Chip ID 00 Region 3
[    5.644650] random: crng init done
[    5.648087] random: 24 urandom warning(s) missed due to ratelimiting
[    6.208007] mt798x-wmac 18000000.wifi: HW/SW Version: 0x8a108a10, Build Time: 20221012174648a
[    6.208007] 
[    6.314260] mt798x-wmac 18000000.wifi: WM Firmware Version: ____000000, Build Time: 20221012174725
[    6.383540] mt798x-wmac 18000000.wifi: WA Firmware Version: DEV_000000, Build Time: 20221012174937
[    6.529961] mt798x-wmac 18000000.wifi: eeprom load fail, use default bin
[    6.540021] mt798x-wmac 18000000.wifi: registering led 'mt76-phy0'
[    6.548161] mt798x-wmac 18000000.wifi: registering led 'mt76-phy1'
[   10.387765] PPP generic driver version 2.4.2
[   10.392753] PPP MPPE Compression module registered
[   10.398179] NET: Registered PF_PPPOX protocol family
[   10.406352] kmodloader: done loading kernel modules from /etc/modules.d/*
[   12.402313] mtk_soc_eth 15100000.ethernet eth0: Link is Down
[   12.483363] mtk_soc_eth 15100000.ethernet eth0: configuring for fixed/2500base-x link mode
[   12.491819] mtk_soc_eth 15100000.ethernet eth0: Link is Up - 2.5Gbps/Full - flow control rx/tx
[   12.495718] device eth0 entered promiscuous mode
[   12.505597] mt7530-mdio mdio-bus:1f lan1: configuring for phy/gmii link mode
[   12.513432] IPv6: ADDRCONF(NETDEV_CHANGE): eth0: link becomes ready
[   12.519996] br-lan: port 1(lan1) entered blocking state
[   12.525222] br-lan: port 1(lan1) entered disabled state
[   12.531930] device lan1 entered promiscuous mode
[   12.660192] mt7530-mdio mdio-bus:1f lan2: configuring for phy/gmii link mode
[   12.670118] br-lan: port 2(lan2) entered blocking state
[   12.675348] br-lan: port 2(lan2) entered disabled state
[   12.682458] device lan2 entered promiscuous mode
[   12.781396] mt7530-mdio mdio-bus:1f lan3: configuring for phy/gmii link mode
[   12.791227] br-lan: port 3(lan3) entered blocking state
[   12.796447] br-lan: port 3(lan3) entered disabled state
[   12.803488] device lan3 entered promiscuous mode
[   12.891837] mt7530-mdio mdio-bus:1f lan4: configuring for phy/gmii link mode
[   12.901655] br-lan: port 4(lan4) entered blocking state
[   12.906875] br-lan: port 4(lan4) entered disabled state
[   12.914087] device lan4 entered promiscuous mode
[   13.217627] mtk_soc_eth 15100000.ethernet eth1: PHY [mdio-bus:06] driver [RTL8221B-VB-CG 2.5Gbps PHY] (irq=POLL)
[   13.227803] mtk_soc_eth 15100000.ethernet eth1: configuring for phy/2500base-x link mode
[   14.689967] IPv6: ADDRCONF(NETDEV_CHANGE): dae0peer: link becomes ready
[   14.696728] IPv6: ADDRCONF(NETDEV_CHANGE): dae0: link becomes ready
[   15.092598] br-lan: port 5(phy1-ap0) entered blocking state
[   15.098225] br-lan: port 5(phy1-ap0) entered disabled state
[   15.103994] device phy1-ap0 entered promiscuous mode
[   15.109181] br-lan: port 5(phy1-ap0) entered blocking state
[   15.114751] br-lan: port 5(phy1-ap0) entered forwarding state
[   15.120734] IPv6: ADDRCONF(NETDEV_CHANGE): br-lan: link becomes ready
[   15.352896] br-lan: port 6(phy0-ap0) entered blocking state
[   15.358505] br-lan: port 6(phy0-ap0) entered disabled state
[   15.364318] device phy0-ap0 entered promiscuous mode
[   15.369516] br-lan: port 6(phy0-ap0) entered blocking state
[   15.375093] br-lan: port 6(phy0-ap0) entered forwarding state
[   15.684901] br-lan: port 5(phy1-ap0) entered disabled state
[   15.691729] br-lan: port 6(phy0-ap0) entered disabled state
[   15.698015] device phy0-ap0 left promiscuous mode
[   15.702844] br-lan: port 6(phy0-ap0) entered disabled state
[   15.952286] mt7530-mdio mdio-bus:1f lan2: Link is Up - 1Gbps/Full - flow control off
[   16.108399] br-lan: port 2(lan2) entered blocking state
[   16.113635] br-lan: port 2(lan2) entered forwarding state
[   16.121154] br-lan: port 6(phy0-ap0) entered blocking state
[   16.126745] br-lan: port 6(phy0-ap0) entered disabled state
[   16.132604] device phy0-ap0 entered promiscuous mode
[   16.408289] mtk_soc_eth 15100000.ethernet eth1: Link is Up - 1Gbps/Full - flow control off
[   16.416644] IPv6: ADDRCONF(NETDEV_CHANGE): eth1: link becomes ready
[   22.808529] pppoe-wan: renamed from ppp0
[   27.237676] IPv6: ADDRCONF(NETDEV_CHANGE): phy1-ap0: link becomes ready
[   27.244453] br-lan: port 5(phy1-ap0) entered blocking state
[   27.250072] br-lan: port 5(phy1-ap0) entered forwarding state
[   29.158791] Ignoring NSS change in VHT Operating Mode Notification from 8a:90:ff:89:fe:e4 with invalid nss 1
[   29.176365] Ignoring NSS change in VHT Operating Mode Notification from 8a:90:ff:89:fe:e4 with invalid nss 2
[   29.551211] IPv6: ADDRCONF(NETDEV_CHANGE): phy0-ap0: link becomes ready
[   29.567963] br-lan: port 6(phy0-ap0) entered blocking state
[   29.573539] br-lan: port 6(phy0-ap0) entered forwarding state
[   39.049175] Ignoring NSS change in VHT Operating Mode Notification from 58:85:e9:f6:9d:4d with invalid nss 1
[   39.071377] Ignoring NSS change in VHT Operating Mode Notification from 58:85:e9:f6:9d:4d with invalid nss 2
[   69.488313] pppoe-wan: renamed from ppp0
[  108.112226] pppoe-wan: renamed from ppp0
[  152.595636] pppoe-wan: renamed from ppp0

**Reply by 1715173329** [reply]
光猫拨号为什么是 pppoe 的日志？你这都是重启之后的日志了，没有意义。pppoe 断开可以调高 lcp echo 阈值。
关闭 dae 后再继续观察问题。
